django/contrib/sessions/backends/cached_db.py

```diff
@@ -57,10 +57,11 @@ def save(self, must_create=False):
             self._cache.set(self.cache_key, self._session, self.get_expiry_age())
         except UpdateError:
             raise
-        except Exception:
-            # import logging
-            # log the error
-            pass
+        except Exception as e:
+            import logging
+
+            logger = logging.getLogger("django.request")
```

Comment:
Please move the logger to module level underneath imports. This is the typical convention in Python code and is the more common approach in Django (see other getLogger() calls) … the exception tends to be for security loggers.

Reply:
Agreed.